### PR TITLE
bypass apprrmor if not enabled, needed for packet 1604

### PR DESCRIPTION
**Pull request by zehicle**
Packet disables apparmor on their 1604 install and the tasks were failing.

### CODE REVIEW

File: deploy/tasks/base-ubuntu1604.yml [hunk 1]
```diff
@@ -30,9 +30,22 @@
...
+    command: service apparmor status
+    register: apparmor_status
+    failed_when: false
+    changed_when: false
+  - name: Detect if AppArmor is working.
+    command: service apparmor status
+    register: apparmor_status
+    failed_when: false
+    changed_when: false
+  - debug: var=apparmor_status
```
Comment (VictorLowther) [inline deploy/tasks/base-ubuntu1604.yml]: 2 of the same command?

**Comment by zehicle** [comment]
this is not working like I wanted.  I'll fix

**Comment by zehicle** [comment]
validated on fresh system w/o the extra code

**Comment by galthaus** [comment]
What did you validate?  That it fails without the new code.   That the new
code fixes.  I'm confused.

**Comment by zehicle** [comment]
I validated that it get's past the issue on packet.  I'm going to validate on AWS right now.

**Comment by zehicle** [comment]
I've been able to run this on both Packet and AWS with the changes.  For Packet, it skips Apparmour.  For AWS, the original code runs (RC=0 there).

**Pull request merged**